collectors/ebpf.plugin/ebpf.d/sync.conf

```diff
@@ -12,3 +12,12 @@
     ebpf load mode = entry
     apps = yes
     update every = 2
+
+# List of monitored syscalls 
+[syscalls]
+    sync = yes
+    msync = yes
+    fsync = yes
+    fdatasync = yes
+    syncfs = yes
+    sync_file_range = yes
```

Comment:
As far as you added configuration for every syscall, you shouldn't create/update dimensions for disabled syscalls.

Reply:
You are right, I brought two helpers that are fixing this.